__test__/itembuilder.test.ts

```diff
@@ -19,10 +19,8 @@ describe("Test ItemBuilder", () => {
 
     test("Create Item with minimum required fields", () => {
         const newItem = new ItemBuilder()
-            .setVault(VAULT_ID)
             .setCategory(CategoryEnum.Login)
             .build();
-        expect(newItem.vault).toEqual({id: VAULT_ID});
```

Comment:
It would be good to make the `expect()` here check that the vault ID is undefined - like we do a couple lines below for `newItem.id`

Reply:
This is a nice idea.
Adjusted.